test/component/uk/gov/hmrc/individualsbenefitsandcreditsapi/stubs/BaseSpec.scala

```diff
@@ -33,15 +34,15 @@ import unit.uk.gov.hmrc.individualsbenefitsandcreditsapi.service.ScopesConfig
 import scala.concurrent.duration.Duration
 
 trait BaseSpec
-    extends FeatureSpec
+    extends AnyFeatureSpec
     with BeforeAndAfterAll
     with BeforeAndAfterEach
     with Matchers
     with GuiceOneServerPerSuite
     with ScopesConfig
     with GivenWhenThen {
 
-  override lazy val port = 9000
+  //override lazy val port = 9000
```

Comment:
think we can just remove this?
```suggestion
```

Reply:
Yup. Just remove it as its redundant. Good job 👍